### PR TITLE
Makes nutrition level show up on self-inspect

### PR DESCRIPTION
**Pull request by Mickyan**
:cl: Mickyan
tweak: You can now see how well fed you are by self-inspecting.
/:cl:

Player feedback for nutrition has always been very poor. Junk food was specifically balanced to only be efficient at higher nutrition levels and overeating comes with its drawbacks, but never know where you stand on nutrition until you either go hungry or overeat. 
Now you can better pace yourself by self-inspecting to have a general idea of how much you should be eating.

### CODE REVIEW

**Comment by JStheguy** [comment]
Don't we already have a UI element that pops up to indicate when you're getting hungry or when you've eaten way too much? Just expand that to include indicating when you're well fed.

**Comment by Mickyan** [comment]
No reason to add more screen clutter, those work just fine as alerts you dont need to constantly check your food level

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -718,6 +718,20 @@
 						to_chat(src, "<span class='danger'>Your thinking is clouded and distant.</span>")
 					else if(oxyloss > 30)
 						to_chat(src, "<span class='danger'>You're choking!</span>")
+
+			if(nutrition > NUTRITION_LEVEL_FULL)
```
Comment (ninjanomnom) [inline code/modules/mob/living/carbon/human/human_defense.dm]: you should be using a switch statement for this

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by YakumoChen** [comment]
I don't see the point, there are already many indicators to when you've eaten too much or are hungry.

**Comment by Alomga** [comment]
What if we give this only to people with the self-aware trait?

**Comment by Mickyan** [comment]
@Alomga I considered it but it's such a basic quality of life thing and self aware is already very useful as it is

File: code/modules/mob/living/carbon/human/human_defense.dm (hunk 1 above)
Comment (Mickyan) [reply code/modules/mob/living/carbon/human/human_defense.dm]: Still learning byond code but I think I got it

**Review by ninjanomnom — APPROVED** [review]

**Comment by Bawhoppen** [comment]
Not all information needs to be immediately available, nor is QoL an argument in ANY circumstance.

**Comment by PKPenguin321** [comment]
>Not all information needs to be immediately available

Fair, but why shouldn't this one specific piece of information be available?

**Comment by Mickyan** [comment]
> nor is QoL an argument in ANY circumstance.

This is possibly the most antithetical statement to good game design I've ever read and it makes me immeasurably angry

**Pull request merged**